internal/controller/instaslice_daemonset.go

```diff
@@ -773,31 +773,21 @@ func (r *InstaSliceDaemonsetReconciler) patchNodeStatusForNode(ctx context.Conte
 	}
 
 	// Patch the node capacity with total GPU memory
-	if err := r.patchNodeStatus(ctx, node, strconv.Itoa(totalMemoryGB)+"Gi"); err != nil {
-		return err
-	}
-
-	return nil
-}
-
-// patch node with accelerator memory capacity
-func (r *InstaSliceDaemonsetReconciler) patchNodeStatus(ctx context.Context, node *v1.Node, memory string) error {
-	logger := logr.FromContext(ctx)
 
 	// Create patch data for accelerator-memory-quota
+	memory := strconv.Itoa(totalMemoryGB) + "Gi"
```

Comment:
resource.MustParse(fmt.Sprintf("%vGi", totalMemoryGB))

Reply:
K8S resource.Quantities do very specific rounding... I think this will prevent issues in the future.

Reply:
done